src/design-tokens/tier-1-definitions/fonts.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;600&display=swap');
```

Comment:
This used to be in the mixins file but not opposed to having it here... any reasons why?

Reply:
I think it was in the mixin file before probably b/c of how we let apps handle bundling. it'd do the right thing as it dynamically built stuff. 

I think it probably should have always been adjacent to the other font file stuff, from what I'm seeing?

Reply:
Makes sense to me!